### DISCUSSION TITLE
To do list display issue in the desktop

### DISCUSSION THREAD

**Discussion by mystic-githuber** [standardnotes/forum#2925] [General]
Currently there is an issue in displaying the to-do list properly on the desktop. Image has been attached below.

When I start typing in the empty field, the line gets fixed.

![Screenshot from 2023-01-02 07-58-05](https://user-images.githubusercontent.com/66426048/210194280-e97696dd-a3cc-4fd7-9f7a-79d8bb8fa457.png)

**Comment by effieeee** [comment]
Hi @mystic-githuber, may I know what version of the desktop app you're using at the moment? Does this also appear to happen when the window is at a specific size, or if the app is in full screen?

**Comment by mystic-githuber** [comment]
@effieeee 
Sorry if it wasn't clear, I meant it was on the browser.  Its fully maximized.

**Comment by KMnO4s** [comment]
I have the exact same issue as the OP, but only in my Firefox desktop browser (Windows 11, fully maximized). I don't have this issue on the desktop app, the Android app or the Edge browser on my work laptop.

**Comment by mystic-githuber** [comment]
@effieeee 

Actually I am using Firefox also in Pop OS.
